app/views/users/show.html.erb

```diff
@@ -1,2 +1,43 @@
-<h1>users</h1>
-<p> Shows info of a single users</p>
\ No newline at end of file
+<div class='center'>
+    <div class="flex">
```

Comment:
- [ ] Please add a pagination button as in the design
![image](https://user-images.githubusercontent.com/74744787/218282935-61ceccae-ea34-4462-80a2-68b3e6133fca.png)